website/docs/guides/importing_resources.html.markdown

```diff
@@ -0,0 +1,304 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Importing resources"
+sidebar_current: "docs-vcd-guides-importing-resources"
+description: |-
+ Provides guidance to VMware Cloud resources importing
+---
+
+# Importing resources
+
+Supported in provider *v3.11+* with Terraform *v1.5.x+* (tested with Terraform v1.5.4).
+
+-> Some parts of this document describe **EXPERIMENTAL** features.
+
+## Overview
+
+Importing is the process of bringing a resource under Terraform control, in those cases where the resource was created
+from a different agent, or a resource was created using Terraform, but some or all of its contents were not.
+
+When we create a resource using Terraform, the side effect of this action is that the resource is stored into [Terraform state][terraform-state],
+which allows us to further manipulate the resource, such as changing its contents or deleting it.
+
+There are cases when we create a resource (for example, a cloned vApp) which has the effect of creating other resources
+(such as VMs) or making them available to another owner (for example, shared catalogs and their contents).
+When we are in such situation, we need to [import][terraform-import] the resource, so that we can handle it with our
+Terraform workflow.
+
+## Basic importing
+
+Up to Terraform 1.4.x, importing meant the conjunction of two operations:
+1. Writing the resource definition into an HCL script
+2. Running the command below, also known as "**the import command**"
+
+```
+terraform import vcd_resource_type.resource_definer path_to_resource
+```
+
+The effect of the above actions (which we can also perform in later versions of Terraform) is that the resource is
+imported into the [state][terraform-state].
+The drawback of this approach is that we need to write the HCL definition of the resource manually, which could result
+in a very time-consuming operation.
+
+## Importing terminology
```

Comment:
I was reading the previous section "Basic importing" and was expecting sections about other "types" of importing to come next, but got "hit" by the "Importing terminology" :) Maybe let's move the "Importing terminology" above the "Basic importing"? Alternatively, move it further down. All in an effort to make the guide even more fluent.

Reply:
moved